torch/csrc/jit/codegen/cuda/test/test_gpu.cpp

```diff
@@ -26125,6 +26125,160 @@ TEST_F(NVFuserTest, FusionTrivialInputForwarding_CUDA) {
   testValidate(fusion, cg_outputs2, {t0, t1}, {t0}, __LINE__, __FILE__);
 }
 
+namespace {
```

Comment:
We should split this file. It is too long that GitHub no longer displays syntax highlights.

Reply:
Is anyone against with just naming test files like `test_gpu1.cpp`, `test_gpu2.cpp`, etc? We should continue have topic-specific test files like `test_gpu_view.cpp`, but there are many that don't belong to one specific category. An easy way is to rename `test_gpu.cpp` to `test_gpu1.cpp` and don't add new tests there anymore. New tests should go into `test_gpu2.cpp` until that file becomes too large, then `test_gpu3.cpp` should be created. Not super neat, but I think it should be just enough.

@zasdfgbnm @csarofeen